README.md

```diff
@@ -331,6 +331,85 @@ personalisation={
 
 </details>
 
+### Precompiled Letter
+
+#### Method
+
+<details>
+<summary>
+Click here to expand for more information.
+</summary>
+
+```python
+response = notifications_client.send_precompiled_letter_notification(
+    reference,      # Reference to identify the notification
+    # content or pdf_file is required
+    content=None,   # base64 encoded string
+    pdf_file=None   # pdf file object
```

Comment:
should this mention that pdf_file is used if provided, else content is used?

and i'd probably push them towards providing a pdf_file over content, because that reduces risk of getting encoding wrong. In fact, is there a requirement to accept base 64 encoded content? the more i think about it the more it feels like it's just an implementation detail

Reply:
OK, lets drop the base64 string